src/vaults/mainnet.json

```diff
@@ -235,6 +235,11 @@
       "slug": "kodiak-wbtc-solvbtc",
       "stakeTokenAddress": "0x3879451f4f69f0c2d37cad45319cff2e7d29c596"
     },
+    {
+      "beraRewardsVault": "0x11fadf69a02340a676b4bd2a27ffd952094fae99",
+      "slug": "kodiak-wbtc-solvbtc",
+      "stakeTokenAddress": "0x3879451f4f69F0c2d37CaD45319cFf2E7d29C596"
```

Comment:
> kodiak-wbtc-solvbtc does not have a token for 0x3879451f4f69F0c2d37CaD45319cFf2E7d29C596

You need to add an entry to `tokens/mainnet.json` with the `address` of `0x3879451f4f69F0c2d37CaD45319cFf2E7d29C596` and all the other details associated.